src/components/sign-up.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import logo from '../assets/logo.svg';
+import facebook from '../assets/facebook.svg';
+import gmail from '../assets/gmail.svg';
+export default function SignUp() {
+  return (
+    <div
+      class="mt-5"
+      style={{
```

Comment:
please use tailwind instead of inline style `className="flex flex-col justify-center"`